backend/app/celery/tasks/job_execution.py

```
@@ -232,6 +232,29 @@ def run_tts_batch_submission(
     )
 
 
+@celery_app.task(bind=True, queue="low_priority", priority=1)
+@gevent_timeout(settings.CELERY_TASK_SOFT_TIME_LIMIT, "run_assessment_pipeline")
+def run_assessment_pipeline(
+    self,
+    run_id: int,
+    organization_id: int,
+    project_id: int,
+    trace_id: str,
+    **kwargs,
```

Ruff (0.15.15)

[warning] Unused function argument: `kwargs`

(ARG001)